### ISSUE TITLE
A couple of drop-down issues.

### ISSUE THREAD

**Issue by mikeamelin** [surveyjs/survey-library#7206]
- [x] something weird with selections

https://github.com/surveyjs/survey-library/assets/17025812/fb6e09e2-c16b-49f5-b514-eb8f3ea8c845

- [x] looks like keyboard navigation doesn't work for auto-generated list 

![Image](https://github.com/surveyjs/survey-library/assets/17025812/0fa02aa2-367c-481e-b417-ef9c58f80260)
![Image](https://github.com/surveyjs/survey-library/assets/17025812/2d0dff5d-2438-486e-9ac7-85e9338af6da)
